logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/LocalNotification.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.notification
+
+import com.wire.kalium.logic.data.id.ConversationId
+
+data class LocalNotificationConversation(
+    val id: ConversationId,
+    val name: String,
```

Comment:
What's this? name of the notification? name of the message sender? name of the group/1on1 conversation?

Reply:
i thought that as far as the data class is called "LocalNotification`Conversation`" it's obviously the conversation name. But I'll rename it